kamon-akka-http/src/main/resources/reference.conf

```diff
@@ -14,6 +14,9 @@ kamon.akka-http {
   # cardinality explosion caused by variable sections in the Path.
   name-generator = default
 
+  # Add http status codes as metric tags. The default value is false
+  http-status-code-as-metric-tag = false
```

Comment:
Maybe add `add-` at the beginning for `add-http-status-code-as-metric-tag`.. that's sort of longish but works